functional-tests/src/test/java/org/infinispan/online/service/caching/CachingServiceTest.java

```diff
@@ -46,6 +47,11 @@ public void should_default_cache_be_accessible_via_hot_rod() throws IOException
       hotRodTester.testBasicEndpointCapabilities(hotRodService);
    }
 
+   @Test(expected = HotRodClientException.class)
+   public void should_default_cache_be_protected_via_hot_rod() throws IOException {
+      hotRodTester.testBasicEndpointCapabilities(hotRodService, false);
```

Comment:
We could implement this exactly the same way as we did with REST:

```
hotRodTester.testIfEndpointIsProtected(hotRodService, false);
```

Reply:
Sure, are you against me adding it to the interface as well?

Reply:
I'm absolutely fine with it!